lib/vfscore/main.c

```diff
@@ -1876,7 +1876,7 @@ int futimesat(int dirfd, const char *pathname, const struct timeval times[2])
 		goto out_errno;
 	return 0;
 
-	out_errno:
+out_errno:
```

Comment:
This PR conflicts with #96 -- let's wait for @razvand to fix this and then you will have to do a force push to fix this.

Reply:
The Pull Request has been updated.